packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -381,12 +387,23 @@ const StaticDesignPicker: React.FC< StaticDesignPickerProps > = ( {
 	purchasedThemes,
 } ) => {
 	const hasCategories = !! categorization?.categories.length;
+	const blankCanvasDesign = {
+		recipe: {
+			stylesheet: 'pub/blank-canvas-blocks',
+		},
+		slug: 'blank-canvas-blocks',
+		title: 'Blank Canvas',
+	} as Design;
+
 	const filteredDesigns = useMemo( () => {
 		const result = categorization?.selection
 			? filterDesignsByCategory( designs, categorization.selection )
 			: designs.slice(); // cloning because otherwise .sort() would mutate the original prop
 
 		result.sort( sortDesigns );
+		isEnabled( 'signup/design-picker-pattern-assembler' ) &&
```

Comment:
I found this a bit hard to read; can we just use a simple `if` statement instead? Also, we can move the `blankCanvasDesign` declaration inside this `if`, because it's only used there.